src/Translator/Translator.php

```diff
@@ -417,7 +417,11 @@ public function translatePlural(
             );
         }
 
-        return $translation[$index];
+        if ($translation[$index] !== '') {
```

Comment:
Maybe better to use empty()

Reply:
`translate()` checks also against the empty string – but check against `null` is missing here 😱

Reply:
empty() will check empty string and null